lib/test/reporters.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const dot = require('internal/test_runner/reporter/dot');
+const spec = require('internal/test_runner/reporter/spec');
+const tap = require('internal/test_runner/reporter/tap');
+
+module.exports = {
+  dot,
+  spec,
+  tap,
+};
```

Comment:
According to the latest performance changes on startup side:

@joyeecheung Would it be useful/impactful to expose these using `exposeLazyInterfaces()` from `lib/internal/util.js`?

Reply:
@anonrig just to make sure I understand correctly, the `target` should be `exports`?
Would it work with the current way the reporters are exported (directly on `module.exports`), or would they need to be changed to named exports?

I assume you mean something similar to:

```javascript
exposeLazyInterfaces(exports, 'internal/test_runner/reporter/dot', ['dot']);
exposeLazyInterfaces(exports, 'internal/test_runner/reporter/spec', ['spec']);
exposeLazyInterfaces(exports, 'internal/test_runner/reporter/tap', ['tap']);
```

Reply:
`exposeLazyInterfaces` does not work for default exports, so we can implement lazy loading the same way as https://github.com/nodejs/node/blob/7c6682957b3c5f86d0616cebc0ad09cc2a1fd50d/lib/test.js#L20